rust/src/combinators/sequential_compositor/interactive/ffi.rs

```diff
@@ -19,7 +19,19 @@ use crate::{
     ),
     derived_types(QO = "$get_distance_type(output_measure)")
 )]
-/// Construct a queryable that interactively composes interactive measurements.
+/// Construct a Measurement that when invoked,
+/// returns a queryable that interactively composes measurements.
+///
+/// **Composition Properties**
+///
+/// * sequential: all measurements are applied to the same dataset
+/// * basic: the composition is the linear sum of the privacy usage of each query
+/// * interactive: mechanisms can be specified based on answers to previous queries
+/// * compositor: all privacy parameters specified up-front
+///
+/// If the privacy measure supports concurrency,
+/// this compositor allows you to spawn multiple interactive mechanisms
+/// and interleave your queries amongst them.
```

Comment:
I worry a bit whenever things get duplicated. Consider trimming one of these down, referencing the other, and explaining how they are different? (I'm still wondering about how we publish docs for private elements... It does add a lot more for the end user to swim through, and makes it harder to use search to find the right method.)

Reply:
I wrote this PR before you could add annotations to bootstrap to suppress generic types in the codegen.  I've adjusted the PR to apply bootstrap to the original function, which avoids needing multiple copies of the docs.